packages/amplify_datastore/lib/amplify_datastore.dart

```diff
@@ -77,11 +78,8 @@ class AmplifyDataStore extends DataStorePluginInterface {
       } on PlatformException catch (e) {
         if (e.code.contains('AmplifyAlreadyConfiguredException') ||
             e.code.contains('AlreadyConfiguredException')) {
-          throw const AmplifyAlreadyConfiguredException(
-            AmplifyExceptionMessages.alreadyConfiguredDefaultMessage,
-            recoverySuggestion:
-                AmplifyExceptionMessages.alreadyConfiguredDefaultSuggestion,
-          );
+          logger.info('Ignoring Amplify Native already configured exception');
+          return;
```

Comment:
Is it correct to return? I think Hub listeners and statements like `NativeApiPlugin.setup` must be set up in the case of a hot restart.